site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -609,6 +609,7 @@ const ReviewDisruption = ({
                                             disruption={disruption}
                                             deleteActionHandler={deleteActionHandler}
                                             isTemplate={disruption.template}
+                                            showChangeLink={true}
```

Comment:
Also, I think it would be a good idea in the publish-edit.api.ts to do a check to ensure an operator user cannot edit a disruption that isn't their own. 

Add the below on line 81 in the publish-edit.api.ts
```
if (session.isOperatorUser && session.operatorOrgId !== validatedDisruptionBody.data.createdByOperatorOrgId) {
            logger.error(
                `Operator with operatorOrgId: ${session.operatorOrgId} is not authorised to edit disruption ${validatedBody.data.disruptionId}`,
            );
            redirectTo(res, ERROR_PATH);
            return;
        }
```
And add this: `createdByOperatorOrgId: z.string().uuid().optional(),` to the disruptionInfoSchema (line 251 in disruptionTypes.zod.ts)

I'd probably add a test case to the publish-edit api too.

Reply:
done